src/redux/actions/genes/loadMarkerGenes.js

```diff
@@ -4,18 +4,19 @@ import {
 
 import { fetchCachedWork } from '../../../utils/cacheRequest';
 
-const loadMarkerGenes = (experimentId, resolution) => async (dispatch, getState) => {
+const loadMarkerGenes = (
+  experimentId, resolution, plotUuid, numGenes = 5,
```

Comment:
Yeah but if you look at `markerGenesLoaded.js` you can see that the genes are put under expression.views[plotUuid] thats where selectedGenes are stored. If we have another markerheatmap plot it will have different selectedGenes depending on what the user chose.

Reply:
or should this still be plotType?